iceoryx_hoofs/include/iceoryx_hoofs/posix_wrapper/named_pipe.hpp

```diff
@@ -67,7 +67,7 @@ class NamedPipe : public DesignPattern::Creation<NamedPipe, IpcChannelError>
 
     /// @brief for compatibility with IpcChannelError
     /// @return always false
-    cxx::expected<bool, IpcChannelError> isOutdated() noexcept;
+    static cxx::expected<bool, IpcChannelError> isOutdated() noexcept;
```

Comment:
@elBoberido here the same issue `isOutdated` is always false for a named pipe and the function never accesses the underlying object therefore it could remain static.

Reply:
@elfenpiff as already mentioned, this breaks the API since it can be used as `Foo::bar` but is only expected to be used as `foo.bar`